### PR TITLE
Create manual-testing document

### PR DESCRIPTION
**Pull request by SimonBasca**
*Thank you for submitting a pull request, your contributions are greatly appreciated!*

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Please make sure you've run the test scheme and all the tests pass. If your changes affect existing tests please make sure to update the tests as well.

### CODE REVIEW

File: Docs/manual-testing [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+   * Exploratory testing
+   * Bug triage
+* **Specific Nightly testing tasks**:
+   * Feature testing
+   * Test case creation
+   * Bug verification
+* **Feature coverage**: yes
+* **Bug verification coverage**: yes
+
+## Release
```
Comment (isabelrios) [inline Docs/manual-testing:47]: Not sure about this release title again... should it be Detailed information about the task performed, and then below the Device coverage and the rest, Full-funcional and the other titles?

Comment (SimonBasca) [reply Docs/manual-testing:47]: The main idea was to have the "Detailed information about the task performed" as a different section and the other ones as sub sections.

File: Docs/manual-testing [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    * Verify the steps provided in the description on an affected build, in order to reproduce the bug (if it wasn't earlier) and on the build that contains the patch, to confirm the fix.
+
+#### Search testing
+
+   * Duration: 1 day
+   * Frequency: RC
+   * Description
+      * Set of tests that cover the interaction of users with URL bar, search engines & search codes (VPN).
+
+#### Bug Verification
```
Comment (isabelrios) [inline Docs/manual-testing:98]: Nit: Is this dup with L83?

**Review by isabelrios — APPROVED** [review]

File: Docs/manual-testing (hunk 2 above)
Comment (SimonBasca) [reply Docs/manual-testing]: Changed to "Bug Triage".

**Pull request merged**